spec/game_spec.rb

```diff
@@ -0,0 +1,116 @@
+require './lib/game.rb'
+
+describe Game  do
+
+    describe 'score' do
+        it 'adds the scores of different rolls, and returns a score' do
+        subject.roll(1)
+        subject.roll(2)
+        expect(subject.score).to eq 3
```

Comment:
Rather than accessing the value of the internal attribute `score` (exposed by an attr_reader/accessor), prefer to assert on the return value of `roll`, since this is the behaviour of the method we're looking to test